core/templates/dev/head/pages/exploration-player-page/layout-directives/learner-local-nav.directive.ts

```diff
@@ -71,8 +71,8 @@ angular.module('oppia').directive('learnerLocalNav', [
           };
           ctrl.showFlagExplorationModal = function() {
             $uibModal.open({
-              templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-                '/pages/exploration-player-page/templates/' +
+              template: require(
```

Comment:
@vojtechjelinek -- in some places, you use "./name-of-template" and in some other places you omit the leading "./". Could we please maintain a consistent convention throughout? I think I prefer the "./name-of-template" one as it is more explicit.

Reply:
Hmm, `./template.html` is used for directives because the TS and HTML files are in the same folder, `smth/template.html` is used for modals because these are not located in the same folders. I can change this but I would go for the `smth/template.html` but I also think that it is okay to keep this as an exception if the HTML file is in the same folder.

Reply:
Oh, I see. I think I would agree with you on going for `smth/template.html` in preference to the other one.

With regards to whether the current hybrid system is OK, I'll defer to you on that. My preference is to have everything be of the form  `smth/template.html` for consistency and to avoid confusion over what "./" refers to.

But I'll defer to you to make the call (if you decide to make them consistent, please file an issue for this) -- and for now, let's get this PR in. Thanks!